datasets/new_york/pipelines/new_york/pipeline.yaml

```diff
@@ -0,0 +1,606 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    table_id: new_york
+    description: "New York Pipelines"
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: new_york
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "@daily"
+    catchup: False
+    default_view: graph
+
+  tasks:
+    - operator: "GKECreateClusterOperator"
+      args:
+        task_id: "create_cluster"
+        project_id: "{{ var.value.gcp_project }}"
+        location: "us-central1-c"
+        body:
+          name: new-york
+          initial_node_count: 4
+          network: "{{ var.value.vpc_network }}"
+          node_config:
+            machine_type: e2-standard-16
+            oauth_scopes:
+              - https://www.googleapis.com/auth/devstorage.read_write
+              - https://www.googleapis.com/auth/cloud-platform
+
+    - operator: "GKEStartPodOperator"
+      description: "Run New York 311 Service Requests Pipeline"
+      args:
+        task_id: "transform_csv_ny_311_service_requests"
+        name: "311_service_requests"
+        project_id: "{{ var.value.gcp_project }}"
+        location: "us-central1-c"
+        cluster_name: new-york
+        namespace: "default"
+        image_pull_policy: "Always"
+        image: "gcr.io/bigquery-public-data-dev/new_york__run_csv_transform_kub_311_service_requests:latest"
+        env_vars:
+          PIPELINE_NAME: "{{ var.json.new_york.container_registry.ny_311_service_requests.pipeline_name }}"
+          SOURCE_URL: "{{ var.json.new_york.container_registry.ny_311_service_requests.source_url }}"
+          CHUNKSIZE: "{{ var.json.new_york.container_registry.ny_311_service_requests.chunksize }}"
+          SOURCE_FILE: "files/data_{{ var.json.new_york.container_registry.ny_311_service_requests.dataset_id }}_{{ var.json.new_york.container_registry.ny_311_service_requests.destination_table }}.csv"
+          TARGET_FILE: "files/data_output_{{ var.json.new_york.container_registry.ny_311_service_requests.dataset_id }}_{{ var.json.new_york.container_registry.ny_311_service_requests.destination_table }}.csv"
+          PROJECT_ID: "{{ var.value.gcp_project }}"
+          DATASET_ID: "{{ var.json.new_york.container_registry.ny_311_service_requests.dataset_id }}"
+          TABLE_ID: "{{ var.json.new_york.container_registry.ny_311_service_requests.destination_table }}"
+          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
+          TARGET_GCS_PATH: "{{ var.json.new_york.container_registry.ny_311_service_requests.target_path }}"
+          SCHEMA_PATH: "{{ var.json.new_york.container_registry.ny_311_service_requests.schema_path }}"
```

Comment:
These are not container registry variables so remove them under the `container_registry` namespace and namespace them under `311_service_requests`, without the `ny_` prefix`as they're already under the parent namespace `new_york`.
```suggestion
          PIPELINE_NAME: "{{ var.json.new_york.311_service_requests.pipeline_name }}"
          SOURCE_URL: "{{ var.json.new_york.311_service_requests.source_url }}"
          CHUNKSIZE: "{{ var.json.new_york.311_service_requests.chunksize }}"
          SOURCE_FILE: "files/data_{{ var.json.new_york.311_service_requests.dataset_id }}_{{ var.json.new_york.311_service_requests.destination_table }}.csv"
          TARGET_FILE: "files/data_output_{{ var.json.new_york.311_service_requests.dataset_id }}_{{ var.json.new_york.311_service_requests.destination_table }}.csv"
          PROJECT_ID: "{{ var.value.gcp_project }}"
          DATASET_ID: "{{ var.json.new_york.311_service_requests.dataset_id }}"
          TABLE_ID: "{{ var.json.new_york.311_service_requests.destination_table }}"
          TARGET_GCS_BUCKET: "{{ var.value.composer_bucket }}"
          TARGET_GCS_PATH: "{{ var.json.new_york.311_service_requests.target_path }}"
          SCHEMA_PATH: "{{ var.json.new_york.311_service_requests.schema_path }}"
```
Please do the same for the rest of the parameterized variables.

Reply:
Not using ny_ prefix causes issues because when the variables start with a numeric, it breaks.